src/pretix/control/forms/organizer_forms/organizer_form.py

```diff
@@ -22,3 +28,187 @@ def clean_slug(self):
+        label=_("Company or Organization Name"),
+        help_text=_("Enter your organization’s legal name."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    address_line_1 = forms.CharField(
+        label=_("Address Line 1"),
+        help_text=_("Street address or P.O. box."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    address_line_2 = forms.CharField(
+        label=_("Address Line 2"),
+        help_text=_("Apartment, suite, unit, etc. (optional)."),
+        required=False,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    zip_code = forms.CharField(
+        label=_("Zip Code"),
+        help_text=_("Enter your postal code."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    city = forms.CharField(
+        label=_("City"),
+        help_text=_("Enter your city."),
+        required=True,
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+    )
+
+    country = forms.ChoiceField(
+        label=_("Country"),
+        help_text=_("Select your country."),
+        required=True,
+        choices=CachedCountries(),
+        initial="US",
+    )
+
+    preferred_language = forms.ChoiceField(
+        label=_("Preferred Language for Correspondence"),
+        help_text=_("Select your preferred language for all communication."),
+        required=True,
+    )
+
+    tax_id = forms.CharField(
+        label=_("Tax ID (e.g., VAT, GST)"),
+        help_text=_(
+            "If you are located in the EU, please provide your VAT ID. "
+            "Without this, we will need to charge VAT on our services and will not be able to issue reverse charge invoices."
+        ),
+        max_length=255,
+        widget=forms.TextInput(attrs={"placeholder": ""}),
+        required=False,
+    )
+
+    def __init__(self, *args, **kwargs):
+        self.organizer = kwargs.pop("organizer", None)
+        self.warning_message = None
+        super().__init__(*args, **kwargs)
+        selected_languages = [
+            (code, name)
+            for code, name in settings.LANGUAGES
+            if code in self.organizer.settings.locales
+        ]
+        self.fields["preferred_language"].choices = selected_languages
+        self.fields["preferred_language"].initial = self.organizer.settings.locale
+        self.set_initial_data()
+
+    def set_initial_data(self):
+        billing_settings = OrganizerBillingModel.objects.filter(
+            organizer_id=self.organizer.id
+        ).first()
+
+        if billing_settings:
+            for field in self.Meta.fields:
+                self.initial[field] = getattr(billing_settings, field, "")
+
+    @staticmethod
+    def get_country_name(country_code):
```

Comment:
Should move this method to a separate function, because there is no reason to bind it to this class, and moving it out has these benefit:

- To call it, you just write `get_country_name`, instead of `self.get_country_name` (longer).
- Code in other modules can use it.
- Code less nested.